test/sdk/unity/README.md

```diff
@@ -1,5 +1,17 @@
 # Agones Unity Test Project
+
 This project is intended to be used for developing the Agones Unity SDK.
 
 ## Add Agones Unity Test Project to Unity Hub
+
 Contained within this folder is a Unity project. After opening the Unity Hub application find the "Add" button. In the file path dialog, select this directory. You will then be able to use the Unity Editor to run Agones Unity SDK tests!
+
+# Running Tests
+
+This project implements basic tests for the Unity SDK using the [Unity Test Framework](https://docs.unity3d.com/Packages/com.unity.test-framework@1.1/manual/index.html). These tests are PlayMode tests.
+
+To run these tests, open the Test Framework window in the Unity editor using Window > General > Test Framwork. More information [in the unity guide](https://docs.unity3d.com/Packages/com.unity.test-framework@1.1/manual/workflow-run-test.html).
+
+# Connecting to the Agones SDK Server
+
+These tests require a local SDK server to connect to. The easiest way to run a server is by [running it locally](https://agones.dev/site/docs/guides/client-sdks/local/#running-the-sdk-server). The tests expect the CountsAndLists flag to be enabled with a list called "players" and a counter called "rooms".
```

Comment:
I wonder how the developer experience can be best served. Ideally the 'Agones SDK Server requirement for test' is something that the Agones Unity SDK tests can detect and let the user know if it isn't running.

I wonder if this locally running Agones SDK Server is something that can be optionally set up by the tests. I know you can add command line arguments for tests; perhaps a new "--start-local-server" flag can be optionally set, and in C# the command to run the local Agones SDK server can be assembled. What do you think?

---

Can you show what the command(s) are to set up the local SDK server in the way you mean from this last sentence?

Reply:
Running a local server is described well on the linked page. Given that we have no control over each developer's environment, I think it is easiest to simply have them run a local server through whatever process works best for them.